packages/ve-table/src/index.jsx

```diff
@@ -2050,10 +2054,10 @@ export default {
 
         // set scrolling
         setScrolling(tableContainerRef) {
-            if (this.hasFixedColumn) {
```

Comment:
I think it is better if we keep this as is and assign the variable `scrollTop` only if it is needed. What do you think?

Reply:
Sure, changes applied.